engines/gob/detection/tables_gob1.h

```diff
@@ -602,6 +602,44 @@
 	0, 0, 0
 },
 
+// -- Amiga --
+
+{
+	{
+		"gob1",
+		"",
+		AD_ENTRY5s("intro.stk", "c65e9cc8ba23a38456242e1f2b1caad4", 135561,
```

Comment:
Unfortunately the intro.stk and all the disk*.stk files have identical head and tail md5 in the DOS version so this doesn't fix the issue (actually it makes matters worse since the DOS EGA version gets detected as Amiga and won't run).
There are no other files to use for detection so the only (easy) way to solve this is to just mirror the DOS entry so the user can pick DOS or Amiga when adding the game
(tl,dr: just remove the disk*.stk entries and use AD_ENTRY1s)

Reply:
For the Amiga version you have to extract all STK's (intro.stk, disk1.stk, disk2.stk, disk3.stk, disk4.stk) if you play with intro.stk/disk1.stk The Game asks the User for Disk2.stk

Reply:
> For the Amiga version you have to extract all STK's (intro.stk, disk1.stk, disk2.stk, disk3.stk, disk4.stk) if you play with intro.stk/disk1.stk The Game asks the User for Disk2.stk

The detection entries just include the files needed to detect a particular version, of course you still need all the files to play